section3/reflection.md

```diff
@@ -1,13 +1,25 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
+ 
+I didn't notice more information being removed this time, as much as I did in section 2. I think practicing in the evening classes I was more familiar with using terminal and atom and had less difficult for section 3. 
 
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+The first app that came to mind that uses Arrays was the Every Dollar budgeting app. You set different categories and enter a list of your spending that month. Each category is likely an Array that the user is constantly adding to creating a list of what was spent for how much. 
+
 3. In your own words, explain what an Array method is.
 
+An Array method is a function that is going to do something to the array to change it, such as removing items or adding items. It is notated with a period between the Array variable name and the method, such as Array.pop.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+A loop does a task over and over as long as certain criteria is met. Using an array we can do the same one task to a whole list of elements. If we had a list of names and wanted to insert each one to the same place of the same email that would likely use a loop.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+Lets say we are preparing for a birthday party and we need to put goodie bags together for the boys and for the girls. Fairies will go in the girls bags and trucks will go in the boys. We are doing the same action over and over putting the toy in the bag but if the bag is yellow, we'll put the girls toys in, if the bag is orange we'll put the boys toy in. This is just like a loop doing one action/task over and over based on criteria.
+
 6. What questions do you still have about Arrays and loops?
+
+At one point I ran a loop that starting running numbers endlessly and had to quit terminal to get it to stop, which made me wonder could an endless loop ever break somehting because you can't get it to stop or can we always quit a program and it will stop no problem?
```

Comment:
There isn't anything you can really mess up at Turing 😉 You could do some damage on the job but measures will be in place to prevent that, really!